packages/@react-spectrum/labeledvalue/docs/LabeledValue.mdx

```diff
@@ -0,0 +1,151 @@
+with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat).
+
+```tsx example
+<LabeledValue label="File size" value={1.2} formatOptions={{style: 'unit', unit: 'megabyte'}} />
+```
+
+An object with `start` and `end` properties may also be provided to format a numeric range.
+
+```tsx example
+<LabeledValue label="Price range" value={{start: 150, end: 400}} formatOptions={{style: 'currency', currency: 'USD', minimumFractionDigits: 0}} />
+```
+
+### Dates and times
+
+The `value` prop may be set to a JavaScript [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object, or one of the types from [@internationalized/date](../internationalized/date.html) to display a date or time, which is formatted according to the user's locale.
+
+```tsx example
+import {today, getLocalTimeZone} from '@internationalized/date';
+
+<LabeledValue label="Date modified" value={today(getLocalTimeZone()).subtract({weeks: 1})} />
+```
+
+By default, the formatting depends on the type of value provided. Above, a [CalendarDate](../internationalized/date/CalendarDate.html) is provided, so only the date is displayed. To display a time, pass a [Time](../internationalized/date/Time.html) object. You can also provide a [CalendarDateTime](../internationalized/date/CalendarDateTime.html) or [ZonedDateTime](../internationalized/date/ZonedDateTime.html) to display a date with a time.
+
+```tsx example
+import {now} from '@internationalized/date';
+
+<LabeledValue label="Page load time" value={now()} />
+```
+
+An object with `start` and `end` properties may also be provided to format a date or time range.
+
+```tsx example
+import {Time} from '@internationalized/date';
+
+<LabeledValue label="Business hours" value={{start: new Time(8, 30), end: new Time(18)}} />
+```
+
+Custom `formatOptions` can also be provided control the exact date format. This prop is compatible
+with the option parameter of [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat).
+
+```tsx example
+<LabeledValue label="Appointment date" value={new Date(2022, 6, 5)} formatOptions={{dateStyle: 'short'}} />
+```
+
+### Lists
+
+When the `value` prop is set to an array of strings, they are rendered as a comma separated list according to the user's locale.
+
+```tsx example
+<LabeledValue label="Pizza toppings" value={['Pepperoni', 'Pineapple', 'Mushroom', 'Garlic']} />
+```
+
+By default, the list is displayed as a conjunction (an "and"-based grouping of items). This may be changed to a disjunction (an "or"-based grouping), or a pure comma separated list using the `formatOptions` prop. This is compatible with the option parameter of [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat).
```

Comment:
```suggestion
By default, the list is displayed as a conjunction (an "and"-based grouping of items). This may be changed to a disjunction (an "or"-based grouping), or a pure comma-separated list using the `formatOptions` prop. This is compatible with the option parameter of [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat).
```